resources/core/charts/monitoring/charts/alert-rules/README.md

```diff
@@ -52,16 +51,15 @@ data:
     {{- include "unhealthy-pods-rules.yaml.tpl" . | indent 4}}
 {{ end }}
 ```
-Under ```data:``` ``` alert.rules:``` is configured the file, [unhealthy-pods-rules.yaml](templates/unhealthy-pods-rules.yaml), where is created a rule for alerting if a pod is not running. 
+Under ```data:``` ``` alert.rules:``` is configured the file, [unhealthy-pods-rules.yaml](templates/unhealthy-pods-rules.yaml), where is created a rule for alerting if a pod is not running.

```

Comment:
``Under the **data. alert.rules** parameter, there is a configuration of the [unhealthy-pods-rules.yaml](templates/unhealthy-pods-rules.yaml) file, which creates a rule for alerting when a Pod is not running.``

Reply:
changed